s3-s3-replication-sam/example-pattern.json

```diff
@@ -0,0 +1,57 @@
+{
+  "title": "AWS S3 bucket replication",
+  "description": "This pattern sets up replication between S3 buckets.",
+  "language": "Python",
+  "level": "200",
+  "framework": "SAM",
+  "introBox": {
+    "headline": "How it works",
+    "text": [
+      "This pattern sets up S3 replication between two buckets. The replication configuration is set up in the source bucket, and the destination bucket is set up to receive the replicated objects. The replication configuration is set up to replicate all objects in the source bucket to the destination bucket."
+    ]
+  },
+  "gitHub": {
+    "template": {
+      "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/s3-s3-replication-sam",
+      "templateURL": "serverless-patterns/s3-s3-replication-sam",
+      "projectFolder": "s3-s3-replication-sam",
+      "templateFile": "template.yaml"
+    }
+  },
+  "resources": {
+    "bullets": [
+      {
+        "text": "Replicating Objects",
+        "link": "https://docs.aws.amazon.com/AmazonS3/latest/userguide/replication.html"
+      },
+      {
+        "text": "Cross Region Replication",
+        "link": "https://aws.amazon.com/about-aws/whats-new/2019/09/amazon-s3-introduces-same-region-replication/"
```

Comment:
I would prefer a link to the documentation (https://docs.aws.amazon.com/AmazonS3/latest/userguide/replication.html) instead of a 5 year old announcement. The announcement is still valid but I believe the documentation provides much more context directly. What do you think?

Reply:
I have updated this with a Youtube video link, the one that you send is already in the json